src/pages/product-details/comments/comments.js

```diff
@@ -30,7 +30,7 @@ const Comments = () => {
   const styles = useStyles();
   const dispatch = useDispatch();
 
-  const {
+  let {
```

Comment:
use const

Reply:
> use const

The value of the variables is reassigned. That`s why ussing "let".